presto-main/src/test/java/com/facebook/presto/sql/planner/assertions/PlanMatchPattern.java

```diff
@@ -583,20 +583,17 @@ public static PlanTestSymbol anySymbol()
         return new FunctionCallProvider(QualifiedName.of(name), toSymbolAliases(args));
     }
 
-    public static ExpectedValueProvider<FunctionCall> functionCall(
-            String name,
-            Optional<WindowFrame> frame,
-            List<String> args)
+    public static ExpectedValueProvider<FunctionCall> windowFunctionCall(String name, List<String> args)
```

Comment:
I propose that we just move these static methods into FunctionCallProvider and make its constructor private again. Exposing the `boolean isWindowFunction` directly seems like it makes it easy to misuse FunctionCall.

Reply:
In that case, the `FunctionCallProvider` class will have to be made public.

Reply:
I have pushed a fixup commit, making this change. Let me know how that looks.